### PR TITLE
Twitter connector v3.0.0 documentation

### PR DESCRIPTION
**Pull request by GihanAyesh**
## Purpose
The documentation explains how to interact with the Twitter connector v3.0.0.
It contains the information about,

- Twitter Connector Overview
- Creating the Twitter Client ID and Access Token
- Configuring Integration Runtime
- Twitter Connector Example
- Twitter Connector Reference

### CODE REVIEW

File: en/docs/reference/connectors/twitter-connector/twitter-connector-overview.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Twitter Connector Overview
+
+The Twitter Connector allows you to work with Twitter, a social networking site where users broadcast short posts known as Tweets. You can use the Twitter connector to work with Tweets, users, lists, and direct messages. The connector uses the [Twitter API v2](https://developer.twitter.com/en/docs/twitter-api) to interact with Twitter.
+
+!!! info
+    If your Twitter application is using **Twitter API v1.1**, you have to use the [Twitter v1.1 Connector](https://docs.wso2.com/display/ESBCONNECTORS/Twitter+Connector+and+Inbound) to interact with Twitter.
```
Comment (SanojPunchihewa) [inline en/docs/reference/connectors/twitter-connector/twitter-connector-overview.md:6]: Shouldn't the connector version be 2.0.7 here?

File: en/docs/reference/connectors/twitter-connector/twitter-connector-credentials.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+6. After successfully setting up the user authentication you can obtain the client ID of the Twitter app which is used for the Twitter connector configuration.
+  <img src="{{base_path}}/assets/img/integrate/connectors/twitter-connector-clientid.png" title="Client ID" width="800" alt="Client ID" />
+
+7. To obtain the Access Token and Refresh Token, follow the [Twitter OAuth 2 guide](https://developer.twitter.com/en/docs/authentication/oauth-2-0/user-access-token#:~:text=Steps%20to%20connect%20using%20OAuth%202.0). 
+
+!!! info
+    The Twitter access token is valid for 2 hours. The refresh token is valid until a new access token is created from the refresh token.
+
+!!! warning
```
Comment (SanojPunchihewa) [inline en/docs/reference/connectors/twitter-connector/twitter-connector-credentials.md:33]: Do we need to mention this?

File: en/docs/reference/connectors/twitter-connector/twitter-connector-configuration.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+If you are using the **Micro Integrator 4.2.0**, you need to add the following message builder to **`<PRODUCT_HOME>`/conf/deployment.toml** file. You can further refer to the [Working with Message Builders and Formatters](https://ei.docs.wso2.com/en/latest/micro-integrator/setup/message_builders_formatters/message-builders-and-formatters/) and [Product Configurations]({{base_path}}/reference/config-catalog-mi/) documentation.
+
+```toml
+[[custom_message_builders]]
+class="org.wso2.micro.integrator.core.json.JsonStreamBuilder"
+content_type = "application/problem+json"
+```
+
+If you are using **EI 6**, you can enable this property by doing the following Axis2 configurations in the **`<PRODUCT_HOME>`/repository/conf/axis2/axis2.xml** and **`<PRODUCT_HOME>`/repository/conf/axis2/axis2_blocking_client.xml** files.
```
Comment (SanojPunchihewa) [inline en/docs/reference/connectors/twitter-connector/twitter-connector-configuration.md:17]:
```suggestion
If you are using **EI 6.x** version, you can enable this property by doing the following Axis2 configurations in the **`<PRODUCT_HOME>`/repository/conf/axis2/axis2.xml** and **`<PRODUCT_HOME>`/repository/conf/axis2/axis2_blocking_client.xml** files.
```

File: en/docs/reference/connectors/twitter-connector/twitter-connector-credentials.md (hunk 2 above)
Comment (GihanAyesh) [reply en/docs/reference/connectors/twitter-connector/twitter-connector-credentials.md:33]: Changed the warning message

**Review by ruthryi — APPROVED** [review]

**Review by SanojPunchihewa — APPROVED** [review]

**Comment by SanojPunchihewa** [comment]
@GihanAyesh Please send the same changes for the [4.2.0](https://github.com/wso2/docs-apim/tree/4.2.0) branch as well

**Pull request merged**